server/data/prisonApi.ts

```diff
@@ -60,15 +77,20 @@ export class PrisonApi extends RestClient {
   getAgencyDetails(context: string, agencyId: string): Promise<Agency> {
     return this.get<Agency>({
       path: `/api/agencies/${agencyId}?activeOnly=false`,
-      query: context
-  })
+      query: context,
+    })
   }
 
-  getStaffDetails(context: string, staffId: string): Promise<Agency> {
-    return this.get<Agency>({
+  getStaffDetails(context: string, staffId: string): Promise<Staff> {
+    return this.get<Staff>({
       path: `/api/users/${staffId}`,
-      query: context
+      query: context,
     })
   }
 
+  getDetails(prisonerNumber: string): Promise<Offender> {
```

Comment:
You may want to rename this to `getPrisonerDetails()` or something like that, for clarity and also for consistency with `getStuffDetails()`.